src/mpid/ch4/netmod/ofi/ofi_finalize.c

```diff
@@ -7,15 +7,89 @@
 #include "ofi_impl.h"
 #include "ofi_noinline.h"
 
+#define MPIDI_OFI_FLUSH_CONTEXT_ID 0xF000
```

Comment:
Why not use COMM_WORLD?

Reply:
I am following the same convention in `MPIDI_OFI_conn_manager_destroy`. We are essentially sending out-of-band messages, so we need some way for the message not to mis-match with regular messages. For one, we are using a separate context-id space reserved for establishing dynamic processes; for two, we use a high enough count (`0xF000`) to avoid collision with regular dynamic process traffic. Ideally, we may want a reserved space for finalize -- leaving it to do separately.